tests/legacy/test_system_collections.py

```diff
@@ -16,21 +16,17 @@ class TestSystemCollections:
     These validate the system collections functionality in the NI-DAQmx Python API.
```

Comment:
FYI, I double checked that you didn't miss anything.

```
PS D:\dev\nidaqmx-python> poetry run pytest -v -k "not grpc and not library"
================================================= test session starts =================================================
platform win32 -- Python 3.9.13, pytest-7.3.1, pluggy-1.0.0 -- D:\dev\nidaqmx-python\.venv\Scripts\python.exe
cachedir: .pytest_cache
rootdir: D:\dev\nidaqmx-python
configfile: pyproject.toml
testpaths: tests
plugins: cov-4.0.0, mock-3.10.0
collected 660 items / 620 deselected / 40 selected

tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path0] PASSED               [  2%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path1] PASSED               [  5%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path2] PASSED               [  7%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path3] PASSED               [ 10%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path4] PASSED               [ 12%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path5] PASSED               [ 15%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path6] PASSED               [ 17%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path7] SKIPPED (Example...) [ 20%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path8] PASSED               [ 22%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path9] PASSED               [ 25%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path10] PASSED              [ 27%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path11] SKIPPED (Exampl...) [ 30%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path12] PASSED              [ 32%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path13] PASSED              [ 35%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path14] SKIPPED (Exampl...) [ 37%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path15] SKIPPED (Exampl...) [ 40%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path16] SKIPPED (Exampl...) [ 42%]
tests/acceptance/test_examples.py::test___shipping_example___run___no_errors[example_path17] PASSED              [ 45%]
tests/legacy/test_enum_bitfields.py::test_coupling_bitfield PASSED                                               [ 47%]
tests/legacy/test_enum_bitfields.py::test_terminal_configuration_bitfield PASSED                                 [ 50%]
tests/legacy/test_enum_bitfields.py::test_trigger_usage_bitfield PASSED                                          [ 52%]
tests/legacy/test_lib.py::TestLib::test_task_handle_type PASSED                                                  [ 55%]
tests/legacy/test_utils.py::TestUtils::test_basic_flatten_flatten_and_unflatten PASSED                           [ 57%]
tests/legacy/test_utils.py::TestUtils::test_backwards_flatten_flatten_and_unflatten PASSED                       [ 60%]
tests/legacy/test_utils.py::TestUtils::test_empty_flatten_flatten_and_unflatten PASSED                           [ 62%]
tests/legacy/test_utils.py::TestUtils::test_leading_zeros_flatten_and_unflatten PASSED                           [ 65%]
tests/unit/test_task.py::test___init___generated_name_saved PASSED                                               [ 67%]
tests/unit/test_task.py::test___init_with_name___specified_name_saved PASSED                                     [ 70%]
tests/unit/test_task.py::test___init_with_name___create_task_called[False] PASSED                                [ 72%]
tests/unit/test_task.py::test___init_with_name___create_task_called[True] PASSED                                 [ 75%]
tests/unit/test_task.py::test___call_alternate_constructor___create_task_not_called[False] PASSED                [ 77%]
tests/unit/test_task.py::test___call_alternate_constructor___create_task_not_called[True] PASSED                 [ 80%]
tests/unit/test_task.py::test___varying_close_on_exit___close___clear_task_called[False] PASSED                  [ 82%]
tests/unit/test_task.py::test___varying_close_on_exit___close___clear_task_called[True] PASSED                   [ 85%]
tests/unit/test_task.py::test___close_on_exit___context_manager___clear_task_called PASSED                       [ 87%]
tests/unit/test_task.py::test___no_close_on_exit___context_manager___clear_task_not_called PASSED                [ 90%]
tests/unit/test_task.py::test___close_on_exit___leak_task___raises_resource_warning PASSED                       [ 92%]
tests/unit/system/storage/test_persisted_task.py::test___persisted_task___load___specified_name_saved PASSED     [ 95%]
tests/unit/system/storage/test_persisted_task.py::test___persisted_task___load___load_task_called[False] PASSED  [ 97%]
tests/unit/system/storage/test_persisted_task.py::test___persisted_task___load___load_task_called[True] PASSED   [100%]

=================================== 35 passed, 5 skipped, 620 deselected in 10.32s ====================================
```

- test_examples.py: these don't use fixtures. No change.
- test_enum_bitfields.py: these are legacy unit tests. No change.
- test_lib.py: these are legacy unit tests. No change.
- test_utils.py: these are legacy unit tests. No change.
- tests/unit/*: these are unit tests. No change.